molgenis_documentation/technical_documentation.md

```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
 * Contribution policies (if your project accepts outside contributions).
 
 
-# MOLGENIS Technical documentation
+# MOLGENIS user documentation
 ---
+## Introduction
```

Comment:
Yes it is! It ends after the 'Should I use MOLGENIS' part. Getting your own MOLGENIS is the black triangle part :p